baseframe/templates/baseframe/bootstrap3/baseframe.html.jinja2

```diff
@@ -121,6 +121,11 @@
   {%- for asset in config.get('ext_js', []) %}
     <script type="text/javascript" src="{{ asset|ext_asset_url }}"></script>
   {%- endfor %}
+
+  <!-- This block is to include JS assets of the app that are not required on all the pages but has to be included before baseframe bundle(assets "js_all").
```

Comment:
Use Jinja2 comments, not HTML comments. This should not render into the final page. Also use `{#- comment #}` (note the hyphen) so that an empty line is not inserted into the output.

Reply:
Changed to `{#- comment #}`